app/internal/user/disable.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
```

Comment:
Prefer to merge with `enable.py` to `availability.py` and call the functions `disable()` and `enable()` for the sake of simplicity :)